package.json

```diff
@@ -6,8 +6,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "clean": "rimraf node_modules/ dist/ || echo 'Project is already clean'",
     "build": "tsc",
+    "clean": "rimraf node_modules/ dist/ || echo 'Project is already clean'",
```

Comment:
@mdtanrikulu I'd say we should also have a (separate) command here to clean the `docs` directory -> especially since both `docs` and `docs:md` write to the same destination directory

Reply:
Calling `rm -rf docs` or `yarn clean:docs` doesn't really make a difference in a sense of complexity. But if we want everyone in the same page, sure I can add that. Opinions?

Reply:
well, at least it eliminates any doubt